---
title: Research Data
category: Research-Data-Management
layout: default
docs_css: markdown
redirect_from: /Research-Data-Management
---

# Definition of research data
There is no consensus on the definition of research data as they are highly heterogeneous. Thus, the definition can vary considerably depending on the research funder, the scientific discipline or subject, and the research data itself {% cite lindst√§dt_2019 biernacka:2020 voigt_2022 %}. We propose the following definition, based on around 20 others: research data is the collection of digital and non-digital objects (excluding scientific publications) that are generated (e.g. through measurements, surveys, source work), studied and stored during or as a result of scientific research activities. These objects are commonly accepted in the scientific community as necessary for the production, validation and documentation of original research results. In the context of Research Data Management (RDM), research data also includes non-data objects such as software and simulations (see further examples below).

The characteristics of research data depend strongly on the context (i.e. conditions of generation, methods used, perspective) {% cite biernacka:2020 %}. Nevertheless, we can try to classify them as follows:

Primary or raw data is potential information generated by a researcher for the first time during a research project. It is unprocessed, possibly even untouched by human hands, unseen by human eyes, unthought by human minds. It needs to be contextualised to make it accessible to the human audience {% cite pomerantz:2015 darby:n.d. golmand_and_martin:2023 %}. This data can be further categorised as observational (e.g. archaeological samples, brain scans, opinion polls), experimental (e.g. clinical trial data, DNA sequencing or organic material) and simulation, which is the modelling of complex processes (e.g. climate simulations) {% cite darby:n.d. %}.

Secondary data is data compiled from existing sources. This includes derived or compiled data (e.g. corpora, databases created by extracting information from multiple secondary sources) {% cite darby:n.d. %}. 

Then there are processed data (i.e. raw data made useful {% cite golmand_and_martin:2023 %}), analysed data (i.e. processed data that has been interpreted {% cite golmand_and_martin:2023 %}), finalised, published or reference data (i.e. curated data that support your research question  {% cite golmand_and_martin:2023 %}, such as gene banks, national statistical archives {% cite darby:n.d. %}) and information about the means necessary to generate data or replicate results (e.g. computer code, experimental methods) {% cite darby:n.d. %}.

Data is differentiated from information (i.e. processed data that can be consumed by humans), knowledge (i.e. information that has been assimilated by humans) and wisdom (i.e. applied knowledge) (Fig. 1) {% cite gerlich:2023 %}. 

# Common data types in microbiology
Data types in microbiology include the following:
* Crystallographic data
* Geospatial data
* Image data
* Linked genotype and phenotype data
* Linked Open Data (LOD)
* Macromolecular structures (e.g. electron microscopy data)
* Microbiome data (e.g. physical microbiome interactions)
* Nucleic acid sequences (e.g. raw sequencing data (reads or traces), genome assemblies, annotated sequences, sample metadata) such as:
  * DNA sequences
    * (Meta)genomes
    * Genetic polymorphism
    * Genomic features
    * Genomic organization
  * RNA sequences
    * 16S, 18S and ITS ribosomal RNA sequences
    * Functional genomics / gene expression data (e.g. ribosome profiling)
    * RNA-protein interactions
    * Small RNA (sRNA)
    * Transcriptomes
* Protein sequences
  * Protein-protein interactions
  * Proteomes
* Quantitative and predictive food microbiology
* Sample and project (meta)data
* Scientific texts
* Semantic data
* Species interaction data (e.g. physical microbial interaction data)
* Standardized bacterial information
* Vertebrate-virus network

{:toc}
